Symfony/CS/Tests/Console/ConfigurationResolverTest.php

```diff
@@ -383,16 +330,12 @@ public function provideResolveConfigByNameCases()
      */
     public function testResolveConfigByNameThatDoesntExists()
     {
-        $this->resolver
-            ->setOption('config', 'NON_EXISTING_CONFIG')
-            ->resolve()
-        ;
+        $this->resolver->setOption('config', 'NON_EXISTING_CONFIG')->resolve();
```

Comment:
I like the previous version better. It was more readable for me.
But maybe just for me ;)